docs/sources/next/javascript-api/k6-experimental/browser/browsercontext/grantpermissions.md

```diff
@@ -17,6 +17,12 @@ Grants specified permissions to the `BrowserContext`. Only grants corresponding
 
 </TableWithNestedRows>
 
+### Returns
+
+| Type            | Description                                                     |
+| --------------- | --------------------------------------------------------------- |
+| `Promise<void>` | A Promise that fulfills when the permissions have been granted. |
+
 ### Example
 
 {{< code >}}
```

Comment:
The examples seems to be missing an `await` in the call to `grantPermissions`.

Reply:
Fixed here https://github.com/grafana/k6-docs/pull/1590/commits/0931b05878c947231f61e5449a6857496513f029